src/app/WriteHandler.h

```diff
@@ -69,9 +69,7 @@ class WriteHandler
      *  @retval #Others If fails to process read request
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR OnWriteRequest(Messaging::ExchangeContext * apExchangeContext, System::PacketBufferHandle aPayload);
-
-    CHIP_ERROR SendWriteResponse(System::PacketBufferHandle aPayload);
```

Comment:
This is removed from the header, but it's still being used in various places and it's still implemented.  What is going on here?

Reply:
we are using CHIP_ERROR WriteHandler::SendWriteResponse() instead of SendWriteResponse(System::PacketBufferHandle aPayload);